lib/capybara_accessibility_audit/audit_system_test_extensions.rb

```diff
@@ -9,6 +9,13 @@ module AuditSystemTestExtensions
       class_attribute :accessibility_audit_after_methods, default: Set.new
       class_attribute :accessibility_audit_enabled, default: true
       class_attribute :accessibility_audit_options, default: ActiveSupport::OrderedOptions.new
+
+      %i[accept_alert accept_confirm dismiss_confirm accept_prompt dismiss_prompt].each do |method|
```

Comment:
[Minor] Do you think this grouping is better?

```suggestion
      %i[accept_alert accept_confirm accept_prompt dismiss_confirm dismiss_prompt].each do |method|
```

Reply:
Unfortunately, `accept_modal` is defined on the Session, but not as part of the [Capybara::DSL](https://github.com/teamcapybara/capybara/blob/0b43cb90ae633ce9b2008e07b997c76833fa570e/lib/capybara/dsl.rb#L49-L55).

The [Capybara::Session](https://github.com/teamcapybara/capybara/blob/0b43cb90ae633ce9b2008e07b997c76833fa570e/lib/capybara/session.rb#L70-L72) defines a `MODAL_METHODS` constant. We can check for that here, then fall back to our own list.